### PR TITLE
Updated the project to work on python 3.4.2

### PR DESCRIPTION
**Pull request by mattBrzezinski**

### CODE REVIEW

File: grouch/ui.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 import logging
 import sys
 from textwrap import TextWrapper
-
+#from bs4 import BeautifulSoup
```
Comment (chris-martin) [inline grouch/ui.py]: What's this comment for?

File: grouch/ui.py [hunk 2]
```diff
@@ -98,7 +98,7 @@ def courses(args, store):
         d = course['description']
         if d is not None:
           for line in wrapper.wrap(d):
-            print line
+            print (line)
```
Comment (chris-martin) [inline grouch/ui.py]: The space before the paren is unconventional - I'd prefer `print(line)` to `print (line)`.

**Comment by chris-martin** [comment]
Thanks - Fix a couple minor style issues and I'll merge it :)

File: grouch/ui.py (hunk 1 above)
Comment (mattBrzezinski) [inline grouch/ui.py]: That was accidentally left in, I was testing the ui.py script and was getting an error about missing the BeautifulSoup module. I didn't realize later that it was throwing the error for a different file.

**Comment by chris-martin** [comment]
Fyi, you can just make additional commits to the branch and the PR will update; don't need to open a new pull request to make revisions.

**Comment by mattBrzezinski** [comment]
I did not know that, thanks for the information!

**Pull request closed without merging**